### ISSUE TITLE
GameGuru Max - Bug - Crouching Increases Fall Velocity

### ISSUE THREAD

**Issue by Lemuripest93** [TheGameCreators/GameGuruRepo#3285]
When you crouch and move off a platform/building etc.. you increase the speed or fall velocity when going down.

Video demonstration: https://www.youtube.com/watch?v=ScIst0GyjQU

To test this yourself, simply build a tower as seen in the video or place a large object, like a skyscraper or platform into your level, spawn on top and then proceed to do a normal jump first as a test, then do the crouch-walk off the edge and find yourself falling at superhuman speeds.

**Comment by Lemuripest93** [comment]
Forgot to mention that the longer you crouch by the edge, seemingly the faster you fall when you move off.

**Comment by Kitakazi** [comment]
Just commenting here since my issue was closed because it was a duplicate. Even though this issue is well over a year old now and seems to have been forgotten it is very much still an issue. So here is my copy/paste from my issue that was closed for being a duplicate. 

If you crouch-walk slowly off a ledge the player will fall down extremely fast compared to normally walking off a ledge. Like the player is instantly magnetized to the ground.

Here is video showing the difference between a normal walk off a ledge and the bug when crouch walking off a ledge:
https://youtu.be/QKk8TsR81WA

To reproduce slowly crouch walk off a high ledge. I set my move speed to 50 in the video. Everything else is default.

Very noticeable when testing my levels as I have times where player crouches through vents and narrow areas and is suppose to fall down into a new room from above. This bug has existed since the start of Max I just didn't get around to reporting it :(

Although extremely rare I have seen this bug happen when crouch walking and coming in contact with a right angles, like the spot where a floor meets a wall where there should be no way the player should be able to pass through them, but the engine instantly tries to pull the player to the ground terrain and pulls the player out of the level completely. 

This happens when the player has been crouched for a long time like Lemuripest93 suggested it seems the longer you crouch the faster you are magnetized to the ground.

**Comment by Lemuripest93** [comment]
@Kitakazi Yeah, quite annoying that this hasn't been fixed yet, i've basically stopped using Max for a while now because of how incomplete it still feels, just toying around with features now and then.

I think this may also be related to this issue #3137.

**Comment by LeeBamberTGC** [comment]
@Lemuripest93 @Kitakazi Fixed for the next EXP build.  It looks like the 3137 is a feature request, so will come back to that after a good chunk of bugs have been squashed. I agree it could be related in part to this one, it's the same area of code in Bullet :)

**Comment by Lemuripest93** [comment]
The newest build seems to have fixed the velocity bug, so i'll close this one for now, would be great if 3137 gets considered as well not too far down the line.

Thanks Lee! :)
